pandas/tests/test_reshape.py

```diff
@@ -443,6 +443,13 @@ def test_simple(self):
         long_frame = wide_to_long(df, ["A", "B"], i="id", j="year")
         tm.assert_frame_equal(long_frame, exp_frame)
 
+    def test_stubs(self):
+        df = pd.DataFrame([[0,1,2,3,8],[4,5,6,7,9]])
```

Comment:
Can you add here the issue number (for which this test was created), so just add `GH9204` as a comment